app/views/signup.erb

```diff
@@ -0,0 +1,19 @@
+<form action = 'signup' method = 'post'>
+  <label for 'name'>
+    please enter your name:
+    <input type = 'text' name = 'name' required>
+  </label>
+  <label for 'email'>
+    your email address:
+    <input type = 'email' name = 'email' value = '<%=@user_email%>' pattern='[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,3}$' required>
+  </label>
+  <label for 'password'>
+    your password:
+    <input type = 'password' name = 'password' required>
+  </label>
+  <label>
+    and please confirm your password:
+    <input type = 'password_confirmation' name = 'password_confirmation' required>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1007202/22468922/2256595a-e7c2-11e6-9f92-40eed10bfe21.png)

Why might this be?